install/updates/73-subid.update

```diff
@@ -16,7 +16,7 @@ default:objectClass: groupofnames
 default:objectClass: nestedgroup
 default:objectClass: top
 default:cn: Subordinate ID Selfservice User
-default:description: User that can self-request subordiante ids
+default:description: User that can self-request subordinate ids
```

Comment:
Hi @rjeffman 
The `default:` keyword does not work for upgrades (according to the ipa-ldap-updater man page, `if the DN of the entry exists then default is ignored`). I believe you can use instead:
```
remove:description: User that can self-request subordiante ids
add:description: User that can self-request subordinate ids
```

Reply:
You probably need the replace syntax. I think an add will try to add the value with every update.

Reply:
@rcritten would it be better to use `default + replace` instead of `remove + add`?

Reply:
I think so.

default kicks in when the entry doesn't already exist. It defines the way something should look brand new. Once the entry exists the default values are all skipped.

One needs to be careful with add because it doesn't, I think, so an existence check so for multi-valued fields it will be appended indefinitely.

A replace is more straightforward but due to the syntax it can break your eyes trying to read it sometimes. And it has the benefit of at most triggering an LDAP mod once. There are a few examples in other update files with replace usage.

Reply:
I updated the PR using `default + replace`.